tests/components/tado/test_climate.py

```diff
@@ -81,7 +81,7 @@ async def test_smartac_with_swing(hass: HomeAssistant) -> None:
         "min_temp": 16.0,
         "preset_mode": "auto",
         "preset_modes": ["away", "home", "auto"],
-        "swing_modes": ["on", "off"],
+        "swing_modes": ["off", "on"],
```

Comment:
Why is this order reversed and what does it do?

Reply:
It's because I changed the order of insertion in swing_modes.
I can reverse it. My bad.